Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/CheckInRepository.kt

```diff
@@ -28,6 +28,11 @@ class CheckInRepository @Inject constructor(
         .allEntries()
         .map { list -> list.map { it.toCheckIn() } }
 
+    suspend fun getCheckInById(checkInId: Long): CheckIn? {
```

Comment:
i also implemented this method in my checkin pr here: https://github.com/corona-warn-app/cwa-app-android/pull/2692/files#diff-c66309f5c643d2c824d912080bb056dadca4875c0347335c7885dfd7c5e139b6R53

we should probably agree on one or the other concept

Reply:
Should looking up an ID throw an exception? :thinking:

Reply:
looking up no, for getting something by id that does not exist i feel it could be valid, but im not saying you should change your implementation to mine, we should just agree on one 😃